libbeat/tests/system/beat/beat.py

```diff
@@ -239,7 +239,9 @@ def read_output_json(self, output_file=None):
                     # hit EOF
                     break
 
-                jsons.append(json.loads(line))
+                event = json.loads(line)
+                del event['@metadata']
```

Comment:
?
I'm removing `@metadata` from events. All outputs but Elasticsearch print the events in JSON-format, as expected by logstash. including `@metadata`. So our beats->LS->ES configs are valid, even if kafka or redis is used. With `@metadata` being somewhat private, we never documented them. That is, I have to remove the fields, as a number of system tests do check all fields in an event are documented.